### PR TITLE
Page macro: RTCIceCandidate value fixes

### PR DESCRIPTION
**Pull request by hamishwillee**
This is part of fixing #3196

This fixes some property values that were imported in place. I think I got it right. Added some notes in place to explain my logic.

### CODE REVIEW

File: files/en-us/web/api/rtcicecandidate/candidate/index.html [hunk 1]
```diff
@@ -82,7 +82,7 @@ <h2 id="Example">Example</h2>
 
 <p>The <code>handleNewIceCandidate()</code> function shown here passes the received
   candidate's SDP text into {{domxref("RTCIceCandidate.RTCIceCandidate",
-  "RTCIceCandidate()")}} to receive an {{domxref("RTCIceCanddiate")}} object in return,
+  "RTCIceCandidate()")}} to receive an {{domxref("RTCIceCandidate")}} object in return,
```
Comment (hamishwillee) [inline files/en-us/web/api/rtcicecandidate/candidate/index.html:85]: FYI only: Typo fix

File: files/en-us/web/api/rtcicecandidate/protocol/index.html [hunk 2]
```diff
@@ -16,13 +16,11 @@
...
 <div>{{APIRef("WebRTC")}}</div>
 
-<p><span class="seoSummary">The <strong>{{domxref("RTCIceCandidate")}}</strong>
-    interface's read-only <strong><code>protocol</code></strong> property is a string
-    which indicates whether the candidate uses {{Glossary("UDP")}} or {{Glossary("TCP")}}
-    as its transport protocol.</span> <code>protocol</code> is <code>null</code> by
-  default if not specified properly in the SDP, but this is an error condition and will
-  result in a thrown exception when you call
-  {{domxref("RTCPeerConnection.addIceCandidate()")}}.</p>
+<p>The <strong>{{domxref("RTCIceCandidate")}}</strong> interface's read-only <strong><code>protocol</code></strong> property is a string
```
Comment (hamishwillee) [inline files/en-us/web/api/rtcicecandidate/protocol/index.html:19]: FYI only: Layout changes only

File: files/en-us/web/api/rtcicecandidate/protocol/index.html [hunk 3]
```diff
@@ -31,16 +29,19 @@
...
+<pre class="brush: js">var <em>protocol</em> = <em>RTCIceCandidate</em>.protocol;</pre>
 
 <h3 id="Value">Value</h3>
 
 <p>A {{domxref("DOMString")}} which indicates what network protocol the candidate uses,
-  UDP or TCP. These values are defined by the enumerated type
-  {{domxref("RTCIceProtocol")}}:</p>
-
-<p>{{page("/en-US/docs/Web/API/RTCIceProtocol", "Values")}}</p>
+  UDP or TCP. These values are defined by the enumerated type {{domxref("RTCIceProtocol")}}:</p>
```
Comment (hamishwillee) [inline files/en-us/web/api/rtcicecandidate/protocol/index.html:37]: @wbamberg Would appreciate your thoughts.

Here I pulled in the content of `RTCIceProtocol` values section. `RTCIceProtocol` is just an enum - to me it feels like we should kind of treat these like dictionaries so I should pull in whatever is useful in that enum and then delete it. Has there been any discussion of that?

File: files/en-us/web/api/rtcicecandidate/tcptype/index.html [hunk 4]
```diff
@@ -29,15 +29,20 @@
...
+<pre class="brush: js">var <em>tcpType</em> = <em>RTCIceCandidate</em>.tcpType;</pre>
 
 <h3 id="Value">Value</h3>
 
-<p>A {{domxref("DOMString")}} whose value is one of those defined by the
-  {{domxref("RTCIceTcpCandidateType")}} enumerated type.</p>
+<p>A {{domxref("DOMString")}} whose value is one of those defined by the {{domxref("RTCIceTcpCandidateType")}} enumerated type.</p>
 
-<p>{{page("/en-US/docs/Web/API/RTCIceTcpCandidateType", "Values")}}</p>
+  <dl>
```
Comment (hamishwillee) [inline files/en-us/web/api/rtcicecandidate/tcptype/index.html:38]: Same approach as above - pulled in `RTCIceTcpCandidateType` enum, but not sure it is useful for that enum to have independent existence (?)

File: files/en-us/web/api/rtcicecandidateinit/candidate/index.html [hunk 5]
```diff
@@ -17,11 +17,43 @@
 ---
 <div>{{APIRef("WebRTC")}}</div>
 
-<p><span class="seoSummary">The optional property <code><strong>candidate</strong></code> in the <strong>{{domxref("RTCIceCandidateInit")}}</strong> dictionary specifies the value of the {{domxref("RTCIceCandidate")}} object's {{domxref("RTCIceCandidate.candidate", "candidate")}} property.</span></p>
+<p>The optional property <code><strong>candidate</strong></code> in the <strong>{{domxref("RTCIceCandidateInit")}}</strong> dictionary specifies the value of the {{domxref("RTCIceCandidate")}} object's {{domxref("RTCIceCandidate.candidate", "candidate")}} property.</p>
 
 <h2 id="Value">Value</h2>
 
-<p>{{page("/en-US/docs/Web/API/RTCIceCandidate/candidate", "Value")}}</p>
+<!-- this is the value that would have been set in /en-US/docs/Web/API/RTCIceCandidate/candidate-->
```
Comment (hamishwillee) [inline files/en-us/web/api/rtcicecandidateinit/candidate/index.html:24]: So [RTCIceCandidate](have https://developer.mozilla.org/en-US/docs/Web/API/RTCIceCandidate) gets seeded from a dictionary called `RTCIceCandidateInit` - which this doc is part of. This dictionary was pulling in the docs for values from `RTCIceCandidate`. To "fix" this I have just copied the content. 

My thinking is that this is OK for removing the macro, but someone should come back and remove that Init dictionary. Right? OK to leave like this now and come back later?

File: files/en-us/web/api/rtcicecandidate/protocol/index.html (hunk 3 above)
Comment (teoli2003) [reply files/en-us/web/api/rtcicecandidate/protocol/index.html:37]: I've done this numerous time, even removing the enum page, as the name of the enum is transparent. Webidl enums are mapped to string litterals.

File: files/en-us/web/api/rtcicecandidateinit/candidate/index.html (hunk 5 above)
Comment (teoli2003) [reply files/en-us/web/api/rtcicecandidateinit/candidate/index.html:24]: Yes, absolutely!

File: files/en-us/web/api/rtcicecandidate/candidate/index.html (hunk 1 above)
Comment (teoli2003) [reply files/en-us/web/api/rtcicecandidate/candidate/index.html:85]: Yep!

File: files/en-us/web/api/rtcicecandidate/protocol/index.html (hunk 2 above)
Comment (teoli2003) [reply files/en-us/web/api/rtcicecandidate/protocol/index.html:19]: Yep!

**Review by teoli2003 — APPROVED** [review]
The actual removal of the enum and the dictionary will happen later. This is already much better.

File: files/en-us/web/api/rtcicecandidateinit/candidate/index.html (hunk 5 above)
Comment (hamishwillee) [reply files/en-us/web/api/rtcicecandidateinit/candidate/index.html:24]: Thanks very much for your advice on this.

File: files/en-us/web/api/rtcicecandidate/protocol/index.html (hunk 3 above)
Comment (wbamberg) [reply files/en-us/web/api/rtcicecandidate/protocol/index.html:37]: Sorry, I am late to this, but I'm +1 on this, FWIW :).

**Pull request merged**